src/olympia/amo/admin.py

```diff
@@ -2,14 +2,36 @@
 import operator
 
 from django.contrib import admin
+from django.contrib.admin.views.main import ChangeList, ChangeListSearchForm, SEARCH_VAR
 from django.core.exceptions import FieldDoesNotExist
 from django.db import models
 from django.db.models.constants import LOOKUP_SEP
 
 from .models import FakeEmail
 
 
+class CommaSearchInAdminChangeListSearchForm(ChangeListSearchForm):
+    def clean(self):
+        self.cleaned_data = super().clean()
+        search_term = self.cleaned_data[SEARCH_VAR]
+        if ',' in search_term:
+            self.cleaned_data[SEARCH_VAR] = ','.join(
+                [term.strip() for term in search_term.split(',') if term.strip()]
```

Comment:
aware we're getting into small nits here, but you don't need the `[...]` for the generator - `string.join` will accept it directly.

Reply:
:+1: I'll fix that in the next PR that also touches this code